README.md

```diff
@@ -23,6 +24,78 @@ You can run the sandbox on your system, or in a virtualbox VM using Vagrant. For
+
+This table contains all parameters and their default values:
+
+| Parameter | Description                                          | Default Value  |
+| --------- | ---------------------------------------------------- | -------------- |
+| ip        | IP of the VM inside the private network              | 192.168.66.211 |
+| cpus      | The amount of available cpus for the virtual machine | 1              |
+| memory    | Maximum memory of the virtual machine                | 1024           |
+| name      | Name of the virtual machine                          | Symfony-CMF VM |
+| hostname  | Hostname of the virtual machine                      | symfony-cmf    |
+
+Unless these parameters are present, the given default value will be used.
+
+You can add a file named *local_machine.yaml* inside the *vagrant/* directory that is able to override these parameters.
+This can be useful if any project member needs another ip as the default ip is reserved by another vm.
+
+#### Puppet configuration
+
+The configuration is done with [Hiera]():
+We have several configuration files inside the *vagrant/hieradata* directory.
+
+Configuration files:
+    - puppet.yaml: This file contains all classes that needs to be included and the environment
+    - common.yaml: Contains all default values
+    - local.yaml: This file is ignored by git and can be used in order to adjust custom parameters
+
+##### Use jackrabbit
+
+By default the Doctrine implementation of Jackalope is used. When using the jackrabbit implementation, you just need to do the following things:
+
+- set the value *cmf::infrastructure::storage::mysql* to false
+- set the value *cmf::infrastructure::storage::jack* to true
+- uncomment the value *jackrabbit::versin*
+- comment all values prefixed with *cmf::infrastructure::phpcr::mysql_wrapper::*
+- run *vagrant provision* (or *vagrant up* if the machine isn't created yet)
+
+Then the jackrabbit-standalone-2.6.0.jar will be downloaded automatically and will be started in the background.
+
+__NOTE__: Please don't change the *jackrabbit::version* parameter unless you've changed the version inside the __jack__ shell script or you'll run into unexpected conflicts!
+
+##### Change the php version
+
+The parameter *cmf::application::php::version* needs to be changed in order to change the version.
+The following values are allowed:
+
+- 5.4
+- 5.5
+- 5.6
+- 7.0
+
+##### Add required packages
+
+The parameter *cmf::infrastructure::packages::installs* contains a list of all packages that will be installed inside the vm.
+
+You simple need to add more package names (or created a new list in the *local.yaml* that will be merged with the values of the *common.yaml*)
+
+##### Changing vhost name
+
+When having multiple projects using the CMF, we need multiple vhost names:
+
+The parameter *cmf::application::symfony::vhost_name* in the file *vagrant/hieradata/common.yaml* needs to be changed in order to change the vhost name.
+
```

Comment:
the problem on windows is that this file is actually immutable.
I should write sth. about this plugin and problems on windows.